ingestion/src/metadata/mixins/pandas/pandas_mixin.py

```diff
@@ -95,7 +95,6 @@ def return_ometa_dataframes_sampled(
                 bucket_name=table.databaseSchema.name,
                 file_extension=table.fileFormat,
             ),
-            is_profiler=True,
```

Comment:
we have removed the argument here and handled everything as part of the datalake util